content/applications/sales/sales/send_quotations/terms_and_conditions.rst

```diff
@@ -1,117 +1,151 @@
+the top of the page. Unpublished pages will **not** be accessible to visitors.
 
-.. image:: terms_and_conditions/terms_conditions_7.png
+Below, is an example of a basic terms and conditions page on an Odoo website:
+
+.. image:: terms_and_conditions/terms-conditions-page-frontend.png
    :align: center
-   :class: img-thumbnail
-   :alt: General Terms & Conditions in your business documents
+   :alt: General Terms and Conditions on a website.
 
-GT&C as attachment in your emails
----------------------------------
+This terms and conditions page can be placed in the footer of all documents, if necessary. To do
+that, access the layout by navigating to :menuselection:`Settings app --> Business Documents
+heading`, and clicking :guilabel:`Configure Document Layout` (located beneath :guilabel:`Document
+Layout`).
 
-Attach an external document with your general terms and conditions when you are about to send your
-quotation by email to your customers.
+Doing so reveals a :guilabel:`Create your document layout` template pop-up form. On this form, in
+the :guilabel:`Footer` field, enter the URL for the terms and conditions page located on the
+website. It's good practice to also provide a brief title prefacing the URL, so customers/clients
+are aware of what the link leads to when they see it on the footer of documents.
 
-.. image:: terms_and_conditions/terms_conditions_5.png
+.. image:: terms_and_conditions/terms-conditions-footer-field.png
    :align: center
-   :class: img-thumbnail
-   :alt: General Terms & conditions as attachment in your email
+   :alt: General Terms and Conditions in business documents.
 
-GT&C as attachment in your quotation templates
-----------------------------------------------
+General terms and conditions as email attachments
+-------------------------------------------------
 
-Create and edit email templates to set a default attachment for all quotation emails that you
-will send in the future. To do so, you have to go to :menuselection:`Sales --> Configuration
---> Quotation templates` and create a new quotation template or modify an existing one. Under
-the confirmation tab, you are now able to activate online signatures, online payments and to set
-a confirmation mail in which you have the possibility to configure the default attachment.
-There, you can put your general terms & conditions.
+Odoo provides the ability to attach an external document, containing general terms and
+conditions, when quotations are sent by email to customers.
 
-.. image:: terms_and_conditions/terms_conditions_6.png
+.. image:: terms_and_conditions/terms-conditions-attach-file.png
    :align: center
-   :class: img-thumbnail
-   :alt: General Terms & conditions as attachment in your quotation templates
+   :alt: General Terms and conditions as attachment in emails.
+
+General terms and conditions as attachments in quotation templates
+------------------------------------------------------------------
```

Comment:
I would move this whole section to be a sub-section beneath "Default terms and conditions on quotation templates", since it's another (and I imagine, less popular) option for configuring terms and conditions on a quotation template.

Also, maybe consider changing the heading to "Terms and conditions as attachments on confirmation emails". I think that's more accurate since the terms and conditions are being sent with the confirmation email, not the quotation, even though it's configured on the quotation template.